app/views/pages/getting_started3.html.erb

```diff
@@ -45,7 +45,6 @@
       <p class="text-center"><%= image_tag 'pages/getting_started/aspects/add-aspect.png' %></p>
       <ol>
         <li><%= t('.add_aspect4').html_safe %></li>
-        <li><%= t('.add_aspect5').html_safe %></li>
```

Comment:
Can you also remove this text from the English translation file please, then it will be removed from WebTranslateIt, so nobody tries to translate it when it isn't used anymore. :)

Reply:
I've just force-pushed a revised commit (I squashed them in to one to make it simpler). Hope all's OK now.